api/python/provisioner/commands/upgrade/set_swupgrade_repo.py

```diff
@@ -272,6 +273,25 @@ def _pre_repo_validation(self, params: inputs.SWUpgradeRepo,
 
         """
         if not params.is_remote():
+            if params.sig_file:
+                logger.info("File with ISO signature is specified. Start GPG "
+                            "signature validation for the ISO")
+                auth_validator = AuthenticityValidator(
```

Comment:
I suggest to consider to call `CheckISOAuthenticity` here instead.

Reply:
Done